website/docs/r/project_api_key.html.markdown

```diff
@@ -35,13 +35,42 @@ resource "mongodbatlas_project_api_key" "test" {
   }
   
   project_assignment {
-    project_id = "74259ee860c43338194b0f8e"
+    project_id = "64229ee820c42228194b0f4a"
     role_names = ["GROUP_READ_ONLY"]
   }
   
 }
 ```
 
+## Example Usage - Create Org PAK and Assign it to Multiple Projects
+
+```terraform
+resource "mongodbatlas_project" "atlas-project" {
+  name   = "ProjectTest"
+  org_id = "60ddf55c27a5a20955a707d7"
+}
+
+resource "mongodbatlas_project_api_key" "api_1" {
+  description = "test api_key multi"
+  project_id  = mongodbatlas_project.atlas-project.id
+
+  project_assignment {
+    project_id = mongodbatlas_project.atlas-project.id
+    role_names = ["ORG_READ_ONLY", "ORG_BILLING_ADMIN", "GROUP_READ_ONLY"]
```

Comment:
Can you clarify if order of elements in `role_names` array matters? if so what does order represent?

Reply:
no, the order does not matter